src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java

```diff
@@ -51,26 +58,51 @@ public EnvelopeCountSummaryReportListResponse getCountSummary(
         );
     }
 
-    @GetMapping(path = "/zip-files-summary")
+    @GetMapping(path = "/zip-files-summary", produces = "application/csv")
     @ApiOperation("Retrieves zip files summary report for the given date and jurisdiction")
-    public ZipFilesSummaryReportListResponse getZipFilesSummary(
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Retrieves zip files summary in csv format.", response = byte[].class),
+    })
+    public ResponseEntity getZipFilesSummary(
         @RequestParam(name = "date") @DateTimeFormat(iso = DATE) LocalDate date,
         @RequestParam(name = "jurisdiction", required = false) String jurisdiction
-    ) {
-        List<ZipFileSummary> result = this.reportsService.getZipFilesSummary(date, jurisdiction);
-        return new ZipFilesSummaryReportListResponse(
-            result
-                .stream()
-                .map(item -> new ZipFilesSummaryReportItem(
-                    item.fileName,
-                    item.dateReceived,
-                    item.timeReceived,
-                    item.dateProcessed,
-                    item.timeProcessed,
-                    item.jurisdiction,
-                    item.status
-                ))
-                .collect(toList())
+    ) throws IOException {
+
+        String fileName = String.format("Zipfiles-report-%s", date.toString());
+        File csvFile = File.createTempFile(fileName, ".csv");
+
+        CSVFormat csvFileHeader = CSVFormat.DEFAULT.withHeader(
+            "Zip File Name",
+            "Date Received",
+            "Time Received",
+            "Date Processed",
+            "Time Processed",
+            "Jurisdiction",
+            "Status"
         );
+
+        List<ZipFileSummaryResponse> result = this.reportsService.getZipFilesSummary(date, jurisdiction);
+
+        FileWriter fileWriter = new FileWriter(csvFile);
+        try (CSVPrinter printer = new CSVPrinter(fileWriter, csvFileHeader)) {
+            for (ZipFileSummaryResponse summary : result) {
+                printer.printRecord(
+                    summary.fileName,
+                    summary.dateReceived,
+                    summary.timeReceived,
+                    summary.dateProcessed,
+                    summary.timeProcessed,
+                    summary.jurisdiction,
+                    summary.status
+                );
+            }
+        }
```

Comment:
perhaps converting to CSV could be a separate component (that can then be tested in separation)

Reply:
moved the csv part to utility class and added tests